src/olympia/accounts/serializers.py

```diff
@@ -170,3 +171,21 @@ def update(self, instance, validated_data):
             # Not .update because some of the instances are new.
             instance.save()
         return instance
+
+
+class LoginUserProfileSerializer(serializers.ModelSerializer):
+    permissions = serializers.SerializerMethodField()
+    picture_url = serializers.SerializerMethodField()
+
+    class Meta:
+        model = UserProfile
+        fields = ('id', 'email', 'name', 'picture_url', 'permissions',
+                  'username')
+        read_only_fields = fields
+
+    def get_permissions(self, obj):
+        return [perm for group in obj.groups_list
```

Comment:
Remove duplicate permissions with a `set()` and `sort()` before returning, to make the output easier to check?